agent/runtime-attach/README.md

```diff
@@ -0,0 +1,23 @@
+# Java runtime attachment for Application Insights
+
+If you can't update the JVM arguments to attach the Application Insights agent for Java (_
+-javaagent:path/to/applicationinsights-agent-3.2.11.jar_), this project allows you to do attachment
```

Comment:
might want to use a placeholder for version. it will require an update each time a newer version of java jar is released.
something like {LATEST_JAVA_AGENT_VERSION}.jar will work.

Reply:
Good point @heyams! Done.